### PR TITLE
add guestion to css

### PR DESCRIPTION
**Pull request by sabovoichita**

### CODE REVIEW

File: src/data/questions/css.js [hunk 1]
```diff
@@ -40,5 +40,31 @@ window.LOAD_QUESTIONS(3, [
         type: "css"
       }
     ]
+  },
+  {
+    id: 3,
```
[the remark this answers is not in the document]
Comment (sabovoichita) [reply src/data/questions/css.js:48]: added

File: src/data/questions/css.js [hunk 2]
```diff
@@ -40,5 +43,298 @@ window.LOAD_QUESTIONS(3, [
...
+}
+ul a {
+  color: green;
+}
+a {
+  color: red;
+}
+</style>  
+
+.....................................................................................................................................................................................
```
Comment (nmatei) [inline src/data/questions/css.js:147]: why so many dots?

File: src/data/questions/css.js [hunk 3]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+       <ul>
+        <li><a href="#" class="example">link</a></li>
+        <li>list item</li>
+        <li>list item</li>
+      </ul>
+    `,
+    answers: [ 
+      "green",
+      "yellow",
+      " blue",
```
Comment (nmatei) [inline src/data/questions/css.js:150]: remove space inside string

File: src/data/questions/css.js [hunk 4]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+        text: `ul li a`,
+        type: "css"
+      }
+    ]
+  },
+  {
+    id: 2,
+    level: 13,
+    text: "Using an attribute selector, how would you select an <a> element with a \"title\" attribute?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:185]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 5]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+        text: `a=title {...}`,
+        type: "css"
+      }
+    ]
+  },
+  {
+    id: 3,
+    level: 13,
+    text: "CSS grid and flexbox are now becoming a more popular way to create page layouts. However, floats are still commonly used, especially when working with an older code base, or if you need to support an older browser version. What are two valid techniques used to clear floats?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:211]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 6]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "Use the overflow property on the floated element or the 'clearfix hack' on either the floated or parent element.",
+      "Use the 'clearfix hack' on the floated element or the overflow property on the parent element.",
+      "Use the 'clearfix hack' on the parent element or use the overflow property with a value other than 'visible.'"
+    ]
+  },
+  {
+    id: 4,
+    level: 13,
+    text: "When adding transparency styles, what is the difference between using the opacity property versus the background property with a rgba() value?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:224]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 7]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "Opacity specifies the level of transparency of an element, including its content. Background with a rgba() value applies transparency to the background color only.",
+      " Opacity applies transparency to the parent and child elements. Background with a rgba() value specifies the level of transparency of the parent element only."
+      
+    ]
+  },
+  {
+    id: 5,
+    level: 13,
+    text: "What is true of block and inline elements? (Alternative: Which statement about block and inline elements is true?)",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:238]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 8]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "A <span> is an example of a block element. <div> is an example of an inline element."
+    ]
+  },
+  {
+    id: 6,
+    level: 13,
+    text: "CSS grid introduced a new length unit, fr, to create flexible grid tracks. Referring to the code sample below, what will the widths of the three columns be?",
+    type: "css",
+    answerType:"radio",
+	  q:`
```
Comment (nmatei) [inline src/data/questions/css.js:253]: 4 spaces instead tab

File: src/data/questions/css.js [hunk 9]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "50px, 200px, 300px",
+      "50px, 100px, 200px",
+      "50px, 50px, 100px"
+    ]
+  },
+  {
+    id: 8,
+    level: 13,
+    text: "What is the use of line-height property?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:287]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 10]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "to control the height of the space between heading elements",
+      "to control the height of the character size",
+      "to control the width of the space between characters"
+    ]
+  },
+  {
+    id: 9,
+    level: 13,
+    text: "Three of these choices are true about class selectors. Which is NOT true?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:300]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 11]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "The same class can be used multiple times per page.",
+      "Class selectors begin with a leading period(dot)",
+      "Classes can be used multiple times per page but not within the same element."
+    ]
+  },
+  {
+    id: 10,
+    level: 13,
+    text: "What is not true about class selectors?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:313]: remove type - you don't have a code question (q - property)

File: src/data/questions/css.js [hunk 12]
```diff
@@ -40,5 +43,293 @@ window.LOAD_QUESTIONS(3, [
...
+      "An element can have multiple class values.",
+      "Class selectors begin with a leading period(dot)",
+      "More than one element can have the same class value."
+        ]
+  },
+  {
+    id: 11,
+    level: 13,
+    text: "There are many properties that can be used to align elements and create page layouts such as float, position, flexbox, and grid. Of these four properties, which one should be used to align a global navigation bar that stays fixed at the top of the page?",
+    type: "css",
```
Comment (nmatei) [inline src/data/questions/css.js:326]: remove type - you don't have a code question (q - property)

**Pull request merged**